torchci/lib/clickhouse.ts

```diff
@@ -1,5 +1,12 @@
+// This file can't be imported by .tsx files due to client/server stuff (ex not
+// being able to import fs, not having env vars). Run `yarn run build` to see
+// the error and where it is imported from if vercel fails to deploy.
 import { createClient } from "@clickhouse/client";
 import { readFileSync } from "fs";
+// Import itself to ensure that mocks can be applied, see
+// https://stackoverflow.com/questions/51900413/jest-mock-function-doesnt-work-while-it-was-called-in-the-other-function
+// https://stackoverflow.com/questions/45111198/how-to-mock-functions-in-the-same-module-using-jest
+import * as thisModule from "./clickhouse";
```

Comment:
TIL!  Previously, I need to refactor the code by moving the function I want to mock to another module instead